### ISSUE TITLE
Failed to initialize geosite

### ISSUE THREAD

**Issue by omidshojaee** [hiddify/hiddify-next#244]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

I do have the latest version of the app and routing assets:

![Screenshot 2023-12-17 110342](https://github.com/hiddify/hiddify-next/assets/31741561/d1fa7962-680b-45f0-9cf7-69801f01cf3a)
![Screenshot 2023-12-17 110357](https://github.com/hiddify/hiddify-next/assets/31741561/165cebbe-23b0-4757-b9cc-476faebb2768)

And I see this error in the log:

![image](https://github.com/hiddify/hiddify-next/assets/31741561/9710ac70-10a2-48e6-8cb5-04103cb4e433)

My region is set to Iran:
![image](https://github.com/hiddify/hiddify-next/assets/31741561/74118a80-3fcb-4e48-a4da-fca2c14a1aa4)

### Reproduce the Bug

Install the latest version and update routing assets then check the logs.

### Version

0.11.1

### Platform/OS

Windows

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by problematicconsumer** [comment]
Different rules (selecting region adds default rules) require different geo assets. In order for these rules to work properly, you should select appropriate geo assets. IR rules are included in [Iran sing-box rules](https://github.com/Chocolate4U/Iran-sing-box-rules) which you have but not selected as active (only one asset can be active at a time and you have default sing-box assets active).
TLDR: select iran-sing-box-rules in routing assets page.
